.gitlab-ci.yml

```diff
@@ -321,7 +321,7 @@ test:rocm_install:
     - $SUDO_CMD apt-get install -y hip-base
     # Install hip-nvcc ignoring dependencies because it depends on cuda metapackage
     # (with heavy libraries, tools etc. that also require GUI and other packages)
-    - apt-get download hip-nvcc
+    - apt-get download hip-nvcc rocm-cmake
```

Comment:
hip-nvcc can be replaced with hip-base. I've been told hip-nvcc is merely a meta package with dependencies on hip-base and cuda.